src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -251,15 +251,17 @@ public static function collectDeprecations($outputFile)
         $deprecations = array();
         $previousErrorHandler = set_error_handler(function ($type, $msg, $file, $line, $context = array()) use (&$deprecations, &$previousErrorHandler) {
             if (E_USER_DEPRECATED !== $type && E_DEPRECATED !== $type) {
-                return $previousErrorHandler ? $previousErrorHandler($type, $msg, $file, $line, $context) : false;
+                // This can be registered before the PHPUnit error handler.
+                if (!$previousErrorHandler) {
+                    $ErrorHandler = class_exists('PHPUnit_Util_ErrorHandler') ? 'PHPUnit_Util_ErrorHandler' : 'PHPUnit\Util\ErrorHandler';
```

Comment:
I think class_exists should be with "false" as second argument

Reply:
Fixed